d3-radar/src/App.js

```diff
@@ -0,0 +1,220 @@
+			}
+			return (
+				{
+					key: name,
+					label: name,
+					values: {
+						...valuesObj
+					},
+				}
+			);
+		})
+		// console.log(variables); // list of spoke variable col names
+		const variableObjList = variables.map((variableName) => {
+			return {
+				key: variableName,
+				label: colInfo[variableName]?.name,
+			}
+		})
+
+		for(let j = 0; j < setCol.length; j++){
+			let thisSet = setCol[j];
+			if (uniqueSets.includes(thisSet)) {
+				for(let setIdx = 0; setIdx < uniqueSets.length; setIdx++){
+					if(setList[setIdx].key === thisSet) {
+						for(let varIdx = 0; varIdx < variables.length; varIdx++){
+							setList[setIdx].values[variables[varIdx]] += data[variables[varIdx]][j];
+						}
+					}
+				}
+			}
+		}
+
+		return {
+			variables: variableObjList,
+			sets: setList,
+		}
+	} else {
+		return (
+			{
+				variables: [
+					{ key: 'resilience', label: 'Resilience' },
+					{ key: 'strength', label: 'Strength' },
+					{ key: 'adaptability', label: 'Adaptability' },
+					{ key: 'creativity', label: 'Creativity' },
+					{ key: 'openness', label: 'Open to Change' },
+					{ key: 'confidence', label: 'Confidence' },
+				],
+				sets: [
+					{
+						key: 'me',
+						label: 'My Scores',
+						values: {
+							resilience: 4,
+							strength: 6,
+							adaptability: 7,
+							creativity: 2,
+							openness: 8,
+							confidence: 1,
+						},
+					},
+					{
+						key: 'everyone',
+						label: 'Everyone',
+						values: {
+							resilience: 10,
+							strength: 8,
+							adaptability: 6,
+							creativity: 4,
+							openness: 2,
+							confidence: 3,
+						},
+					},
+					{
+						key: 'everyone2',
+						label: 'Everyone2',
+						values: {
+							resilience: 1,
+							strength: 6,
+							adaptability: 3,
+							creativity: 1,
+							openness: 6,
+							confidence: 9,
+						},
+					},
+				],
+			}
+		);
+	}
+}
+
+function onlyUnique(value, index, array) {
+	return array.indexOf(value) === index;
+  }
+
+function App() {
+	// useConfig, useElementColumns, useElementData is a function of the Sigma API
+	const config = useConfig(); // all the info of the selections in the LHS panel
+	const columns = useElementColumns(config.source); // name of all cols in "source"
+	const sigmaData = useElementData(config.source); // data in the table
+	const numGroups = config.number || 5;
+
+	const data = useMemo(() => {
+		const sets = config.sets ?? [];
+		const variables = config.variables ?? [];
+
+		return {
+			name: "data",
+			children: chunk(sigmaData, columns, sets, variables, numGroups),
+		};
+	}, [columns, config.sets, config.variables, sigmaData, numGroups]);
+	
+	const realData = data.children;
+
+	const axisMax = .001;
+	return <Radar
+		width={500}
+		height={500}
+		padding={70}
+		domainMax={axisMax}
+		highlighted={null}
+		onHover={(point) => {
+			if (point) {
+				console.log('hello my name is fred');
+			} else {
+				console.log('lol butts');
+			}
+		}}
```

Comment:
Is this needed? Seems inappropriate for plugins that we are sharing with customers.